class.jetpack-client.php

```diff
@@ -216,4 +219,36 @@ public static function set_time_diff( &$response, $force_set = false ) {
 			}
 		}
 	}
+
+	/**
+	 * Query the WordPress.com REST API using the blog token
+	 */
+	static function wpcom_json_api_request_as_blog( $path, $version = self::WPCOM_JSON_API_VERSION, $args = array(), $body = null ) {
+		$filtered_args = array_intersect_key( $args, array(
+			'method'      => 'string',
+			'timeout'     => 'int',
+			'redirection' => 'int',
+		) );
+
+		$proto = apply_filters( 'jetpack_can_make_outbound_https', true ) ? 'https' : 'http';
```

Comment:
@georgestephanis Honestly, I can't remember what `jetpack_can_make_outbound_https` was supposed to do :)

I _think_ I remember someone mentioning (on the API side of the discussion) that not all hosts are capable of outbound https and maybe this was some half formed solution..?